packages/mermaid/src/docs/syntax/gitgraph.md

```diff
@@ -513,18 +513,19 @@ Here, we have changed the default main branch name to `MetroLine1`.
 
 ## Orientation (v10.3.0+)
 
-In Mermaid, the default orientation is Left to Right. The branches are lined vertically.
+In Mermaid, the default orientation is for commits to run from **left to right** and for branches to be stacked on top of one another.
+
+You can set this explicitly with `LR:` after `gitGraph`.
```

Comment:
I think it is better to say which modes are supported from the very beginning

```
Mermaid supports two graph orientations: **Left to Right**(default) and **Top to Bottom**.

You can set this explicitly with `LR:` for **Left to Right** and `TB:` for **Top to Bottom** orientation respectively after `gitGraph`
```

Reply:
Yeah, agreed. I've gone with something like this in my latest commit.